static/skins/content/dark/content.min.css

```diff
@@ -0,0 +1,7 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
```

Comment:
Are you sure this is not done with webpack on building anyways? Then we could remove all the minified css files...

Reply:
no, it isn't done